emission/analysis/classification/inference/labels/inferrers.py

```diff
@@ -6,6 +6,7 @@
 import copy
 import time
 import arrow
+from uuid import UUID
```

Comment:
why do we need this import?

Reply:
Might have added to test locally, removed from latest fixes.